src/private/WRLDRoutingServiceHelpers.mm

```diff
@@ -0,0 +1,80 @@
+#include "WRLDRoutingServiceHelpers.h"
+
+#import "WRLDRoute.h"
+#import "WRLDRouteSection.h"
+#import "WRLDRouteDirections.h"
+#import "WRLDRouteStep.h"
+#import "WRLDRouteTransportationMode.h"
+
+#import <CoreLocation/CoreLocation.h>
+
+namespace Eegeo
+{
+    namespace Routes
+    {
+        namespace Webservice
+        {
+            namespace Helpers
+            {
+                WRLDRoutingQueryResponse* CreateWRLDRoutingQueryResponse(const Eegeo::Routes::Webservice::RoutingQueryResponse& result)
```

Comment:
This method is huge, and building ~5 different object types.  Could do with factoring out into sub methods for constructing each segment. i.e.  createWRLDRoute, createWRLDRouteSection, createWRLDRouteStep etc.

Reply:
WRLDRoutingServiceHelpers refactored according to review.
https://github.com/wrld3d/ios-api/pull/23/commits/15e1aba170bcdc5f315e3a0fe30b386c78bc893f